source/style-guide/style/lists.txt

```diff
@@ -133,31 +137,38 @@ The following sections show examples of the indicated types of lists.
 All List Items Are Sentences
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When you create an isolated network, the following limitations apply:
+When you create an AWS PrivateLink, the following limitations apply:
+
+- You can use AWS PrivateLink to connect to Atlas clusters running
+  MongoDB version 3.6 or later.
+
+- AWS PrivateLink must be active in all regions into which you deploy a
+  multi-region cluster. You receive an error if AWS PrivateLink is
+  active in some, but not all, targeted regions.
 
-- The isolated network must exist in the same region as the server.
-- You can create up to three isolated networks with up to 64 servers
-  attached to each one.
-- After you create an isolated network, you can't rename it.
+- You can do only one of the following:
+
+  - Create more than one private endpoints in a single region
```

Comment:
s/more than one private endpoints/multiple private endpoints

Also, this bullet should end with a period (.)